common.mk

```diff
@@ -49,6 +49,8 @@ ifeq ($(and $(LATEST_TAG),$(IS_TAG)),NO)
 endif
 export VERSION
 
+GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
```

Comment:
Maybe also include a dirty status (e.g. append `*` or `+` if branch is not clean)

Reply:
Not really needed. The software version includes that and is also transmitted to Prometheus as `software_version` label.